vllm/model_executor/models/falcon.py

```diff
@@ -377,10 +377,6 @@ def __init__(
         self.config = config
         self.linear_method = linear_method
         self.transformer = FalconModel(config, linear_method)
-        self.lm_head = ParallelLMHead(
-            config.vocab_size,
-            config.hidden_size,
-        )
```

Comment:
Why is this change needed?

Reply:
Falcon has tied embeddings and some model checkpoints I tested don't contain `lm.head` weight. I agree this should not be part of this PR though.